### PR TITLE
v1.1.3 - Bug Fix release

### PR DESCRIPTION
**Pull request by titom73**
## Change Summary

Release v1.1.3

## Release Notes

- Fix exception for passing empty dict (#500)
- Storm control in l3ls_evpn the unit as optional (#557)
- Fix l3ls role documentation about igmp snooping (#545)
- Include default VRF in mgmt api for eos_l3ls_evpn (#533)
- svi and l2vlan keys to check and tested as int (#536)
- Fix L2 leaves uplinks when MLAG is not desired (#544)
- Change location for MTU knob so SVI 4093 uses it (#526)
- Fix incorrect send_community for underlay on spines (#592)
- Enhance logging event, tacacs timeout and platform sand multicast replication features (#650)
- Fix ansible-lint and ansible-test lint issues (#680)
- Add collection dependencies (#663)
- Code readability pep8 style / issue with Ansible Tower (4603e2b)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md [hunk 1]
```diff
@@ -675,6 +682,8 @@ ethernet_interfaces:
     speed: < interface_speed >
     mtu: < mtu >
     type: < routed | switched >
+        logging_event:
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: indentation looks wrong here

Comment (titom73) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md]: Thanks for the spot. 
It requires to update this element to fit with PR#570 as well (1bb9df3)

**Review by carlbuchmann — APPROVED** [review]
LGTM and validated with avd-validation lab.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**